toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -86,6 +87,14 @@ func buildLiveOSConfig(inputArtifactsStore *IsoArtifactsStore, isoConfig *imagec
 			config.kernelCommandLine = isoConfig.KernelCommandLine
 			config.additionalFiles = isoConfig.AdditionalFiles
 			config.initramfsType = isoConfig.InitramfsType
+			if isoConfig.KdumpBootFiles != nil {
+				switch *isoConfig.KdumpBootFiles {
+				case imagecustomizerapi.KdumpBootFilesTypeNone, imagecustomizerapi.KdumpBootFilesTypeKeep:
```

Comment:
Why not use the `KdumpBootFilesType.IsValid()` function here?

Reply:
pending